packages/alfa-style/src/property/border-block-end-width.ts

```diff
@@ -0,0 +1,67 @@
+import { Keyword, Length } from "@siteimprove/alfa-css";
+import { Parser } from "@siteimprove/alfa-parser";
+
+import { Property } from "../property";
+import { Resolver } from "../resolver";
+import { Value } from "../value";
+
+const { either } = Parser;
+
+/**
+ * @internal
+ */
+export type Specified =
+  | Length
+  | Keyword<"thin">
+  | Keyword<"medium">
+  | Keyword<"thick">;
+
+/**
+ * @internal
+ */
+export type Computed = Length<"px">;
+
+/**
+ * @internal
+ */
+export const parse = either(
+  Keyword.parse("thin", "medium", "thick"),
+  Length.parse
+);
+
+/**
+ * @see https://developer.mozilla.org/en-US/docs/Web/CSS/border-block-end-style
+ * @internal
+ */
+export default Property.of<Specified, Computed>(
+  Length.of(3, "px"),
+  parse,
+  (borderWidth, style) => {
+    if (
+      style
+        .computed("border-block-end-style")
+        .some(({ value }) => value === "none" || value === "hidden")
+    ) {
+      return Value.of(Length.of(0, "px"));
+    }
+
+    return borderWidth.map((value) => {
+      switch (value.type) {
+        case "keyword":
+          switch (value.value) {
+            case "thin":
+              return Length.of(1, "px");
+
+            case "medium":
+              return Length.of(3, "px");
+
+            case "thick":
+              return Length.of(5, "px");
+          }
```

Comment:
Have we checked these in browsers?

Reply:
I copied this over from `outline-width` 🤔 I've just checked and Chrome seems to follow 1/3/5, but not Firefox (shows 0,8/2,4/4,8). https://jsfiddle.net/04jkaeon/

[This](https://drafts.csswg.org/css-backgrounds-3/#border-width) says the values are not defined, so I'm not sure what to do with it.

Reply:
That's mighty fine then 👍